keyform/templates/keyform/home.html

```diff
@@ -1,6 +1,59 @@
 {% extends "keyform/base.html" %}
 {% load i18n %}
+{% load static %}
+{% block css %}
+    {{ block.super }}
+    <link rel='stylesheet' type='text/css' href="{% static 'keyform/css/home.css' %}"/>
+{% endblock %}
 
 {% block content %}
-    {% trans 'Welcome to Keyform!' %}
+    <h2 class="title">{% trans 'Keyform Requests' %}</h2>
+    <div class="object-list">
+        <div class="header row">
+            <div class="large-3 columns text-center">
+                {% trans 'Date Submitted' %}
+            </div>
+            <div class="large-2 columns text-center">
+                {% trans 'Residence Hall' %}
+            </div>
+            <div class="large-2 columns text-center">
+                {% trans 'Room #' %}
+            </div>
+            <div class="large-2 columns text-center">
+                {% trans 'Core #' %}
+            </div>
+            <div class="large-1 columns text-center">
+                {% trans 'Comments' %}
+            </div>
+            <div class="large-2 columns text-center">
+                {% trans 'Status' %}
+            </div>
+        </div>
+        {% for object in object_list %}
+            <div class="object row">
+                <div class="large-3 columns text-center">
+                    {{ object.created_timestamp|date:"m/d/Y" }}
+                </div>
+                <div class="large-2 columns text-center">
+                    {{ object.building }}
+                </div>
+                <div class="large-2 columns text-center">
+                    {% for key in object.keydata_set.all %}
+                        {{ key.room_number }}{% if not forloop.last %}, {% endif %}
```

Comment:
The table is kind of broken if `keydata_set` is empty. You can use the [{% empty %} clause](https://docs.djangoproject.com/en/1.11/ref/templates/builtins/#for-empty) in conjunction with a non-breaking space (`&nbsp;`) to keep this from breaking everything.